### PR TITLE
[*] Generate - Fixing port and authentication

### LINKED ISSUE #21: Custom base application url when using lumber-generate
Hello there !

I can't wait to see this great tool connected on my project :)

lumber-generate ended successfully on my webserver (i'm not working on localhost)

However, I get
"Login
Oops, cannot reach your application. Are you sure it is running?"

because I'm not able to enter my custom base url, for instance
http://www.myproject.com:8080 instead of http://localhost:3000

How can I fix this setting, from https://app.forestadmin.com (or from the command line tool) ?

### PR DESCRIPTION
**Pull request by VincentMolinie**
Fixes #21
Fixes #142 (The other PRr doesn't fix the problem)

### CODE REVIEW

**Comment by VincentMolinie** [comment]
[(?) Lumber - On generate, the app port is not set to env](https://trello.com/c/eEmiX5IU/2672-lumber-on-generate-the-app-port-is-not-set-to-env)

**Comment by VincentMolinie** [comment]
[(?) Lumber - On generate, if an existing user is not logged don't throw an error](https://trello.com/c/Agbcn8lz/2673-lumber-on-generate-if-an-existing-user-is-not-logged-dont-throw-an-error)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Release 1.9.1
+
+### Fixed
+- Generate - App port is set in .env file
+
+
```
Comment (ArnaudValensi) [inline CHANGELOG.md]: To be consistent, we should only put one blank line

File: services/authenticator.js [hunk 2]
```diff
@@ -138,6 +139,14 @@ function Authenticator() {
         return fs.writeFileSync(`${os.homedir()}/.lumberrc`, auth.token);
       });
   };
+
+  this.authenticateAndCreateProject = (config) => {
+    if (config.authToken) {
+      return this.createProject(config);
+    }
+
+    return this.register(config);
```
Comment (ArnaudValensi) [inline services/authenticator.js]: This way it is difficult to understand that the method `register` also create a project. You should either rename the function or do the `createProject` here.

**Comment by ArnaudValensi** [comment]
This PR is not clean, it contains changes that should not be here. It is because you need to rebase it from master. When you want the last changes of the branch you want to merge to, you should not use `git pull` but `git pull --rebase`. https://www.atlassian.com/git/tutorials/merging-vs-rebasing

**Pull request merged**